packages/node_modules/samples/browser-single-party-call/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <title>Sample: Single Party Calling</title>
+  <body>
+    <h1>Single Party Calling</h1>
+    <p>This sample demonstrates 1:1 calling.</p>
+    <p>In case your computer doesn't have a camera attached, you can load this
```

Comment:
Should we instruct how to do the fake media on chrome?

Reply:
i'm actually leaning toward removing it entirely. it's lead to more confusion than i expected and i really only put it in there because i was being lazy while working from my mac mini.